lib/modules/versioning/cargo/index.ts

```diff
@@ -63,7 +62,7 @@ const isLessThanRange = (version: string, range: string): boolean =>
   !!npm.isLessThanRange?.(version, cargo2npm(range));
 
 export const isValid = (input: string): boolean =>
-  npm.isValid(cargo2npm(input));
+  !!input && npm.isValid(cargo2npm(input));
```

Comment:
why this?

Reply:
Sorry, it's a local change that is not supposed to be there. Reverted in fb1274e8c9f8fecbfc80bf5d14c93c35837d260d.